hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/client/CertificateClient.java

```diff
@@ -278,4 +281,14 @@ void storeRootCACertificate(String pemEncodedCert, boolean force)
    */
   long getLatestCrlId() throws IOException;
 
+  default void assertKeyPair() throws OzoneSecurityException {
+    try {
+      Objects.requireNonNull(getPublicKey());
+      Objects.requireNonNull(getPrivateKey());
+      Objects.requireNonNull(getCertificate());
```

Comment:
NIT: can we rename it to like assertValidKeyCerts to reflect the code?

Reply:
Renamed to `assertValidKeysAndCertificate`.